themes/powerline-naked/powerline-naked.base.bash

```diff
@@ -1,7 +1,8 @@
 . "$BASH_IT/themes/powerline/powerline.base.bash"
 
-function __powerline_left_segment {
-  local OLD_IFS="${IFS}"; IFS="|"
+function __powerline_left_segment() {
+  local OLD_IFS="${IFS}"
+  IFS="|"
```

Comment:
Sorry my note was regarding reverting ALL changes in this file as they don't relate to the PR.

Reply:
fixed